test/Microsoft.Azure.EventHubs.UnitTests/EventHubClientTests.cs

```diff
@@ -622,8 +624,8 @@ void ValidateRetryPolicyCustom()
         {
             // Rebuild connection string with a nonexistent entity.
             var conSettings = new EventHubsConnectionStringBuilder(this.connectionString);
```

Comment:
Good catch!

---
In reply to: [83526893](https://github.com/Azure/azure-event-hubs-dotnet/pull/22#discussion_r83526893) [](ancestors = 83526893)